### PR TITLE
fuchsia: Correct PlatformView scale/transform

### PR DESCRIPTION
**Pull request by arbreng**
The new embedder renderer for Fuchsia doesn't handle scale or transform for embedded views correctly, so they get scaled wrong on some devices.

Fixes b/180926644

Tested by flutter-embedder-tests in Fuchsia tree

### CODE REVIEW

File: shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.cc [hunk 1]
```diff
@@ -237,6 +238,23 @@ void FuchsiaExternalViewEmbedder::SubmitFrame(
...
+          const auto& mutator = *i;
+          switch (mutator->GetType()) {
+            case flutter::MutatorType::transform: {
+              mutatorsTransform.preConcat(mutator->GetMatrix());
+            } break;
+            default: {
+              break;
+            }
+          }
+        }
```
Comment (dreveman) [inline shell/platform/fuchsia/flutter/fuchsia_external_view_embedder.cc:255]: Can we move all this into a DCHECK if that's the only use? maybe move it to a `SkMatrix GetMutatorsTransform(const MutatorStack& mutatorStack)` function in anonymous namespace.

**Review by dreveman — COMMENTED** [review]
looks good. what's the current plan for testing this?

**Comment by dreveman** [comment]
lgtm if we have a plan for testing this and making sure it doesn't regress in the future

**Comment by chinmaygarde** [comment]
@arbreng: Any updates on @dreveman's last query? Are there any roadblocks to testing and landing this?

**Comment by jaeheon** [comment]
I'm writing a test for this in fuchsia.git.

**Pull request merged**